cmd/istioctl/main.go

```diff
@@ -299,16 +299,22 @@ istioctl delete route-rule productpage-default
 					c.Println(c.UsageString())
 					return fmt.Errorf("provide configuration type and name or -f option")
 				}
+				var errs error
 				for i := 1; i < len(args); i++ {
 					if err := setup(args[0], args[i]); err != nil {
+						// If the user specified an invalid argument on the CLI,
```

Comment:
s/argument/kind? That might make it more obvious why this case is handled differently than the file case below.